### PR TITLE
Cult balance pass

### PR DESCRIPTION
**Pull request by Robustin**
I tested these changes myself, it is working as intended(tm). 

:cl: Robustin
add: The initial supply talisman can now be used to create runed metal, 5 per use
add: The void torch is now available from the altar, use it to transport any item instantly to any cultist! It appears alongside the veil shifter as part of the "Veil Walker" set. 
add: Cult floors no longer appear on mesons! 
tweak: Flagellant's Robes now increases damage taken by 50%, down from 100%
tweak: Veil Shifter has 4 uses, up from 2
tweak: Shuttle curse now delays shuttle 3 minutes, up from 2.5 minutes
tweak: Cult forges, archives, and airlocks have had their cost reduced by 1 runed metal each
/:cl:

### CODE REVIEW

File: code/game/gamemodes/cult/cult_items.dm [hunk 1]
```diff
@@ -215,7 +215,7 @@
 	flags_inv = HIDEJUMPSUIT
 	allowed = list(/obj/item/weapon/tome,/obj/item/weapon/melee/cultblade)
 	body_parts_covered = CHEST|GROIN|LEGS|ARMS
-	armor = list(melee = -100, bullet = -100, laser = -100,energy = -100, bomb = -100, bio = -100, rad = -100)
+	armor = list(melee = -50, bullet = -50, laser = -100,energy = -50, bomb = -50, bio = -50, rad = -50)
```
Comment (GunHog) [inline code/game/gamemodes/cult/cult_items.dm]: Some of these do not work work at negative values, keep that in mind.

File: code/game/gamemodes/cult/cult_items.dm [hunk 2]
```diff
@@ -215,7 +215,7 @@
...
 	slowdown = -1
 	hooded = 1
 	hoodtype = /obj/item/clothing/head/berserkerhood
```
Comment (ChangelingRain) [inline code/game/gamemodes/cult/cult_items.dm]: still literally no reason to ever put the hood up because it gives you noooothiiiiing

Comment (Robustin) [inline code/game/gamemodes/cult/cult_items.dm]: Covers your face nerd, plus massive style points.

Easily worth it :+1:

**Comment by Niknakflak** [comment]
>tweak: Shuttle curse now delays shuttle 3 minutes, up from 2.5 minutes

Wasn't this denied in another PR

**Comment by Robustin** [comment]
That PR moved the total delay time from 5 minutes to 12 minutes.

This PR moves it from 5 minutes to 6 minutes.

File: code/game/gamemodes/cult/cult_items.dm [hunk 3]
```diff
@@ -295,12 +295,12 @@
 		sleep(20)
 		var/global/list/curses
 		if(!curses)
-			curses = list("A fuel technician just slit his own throat and begged for death. The shuttle will be delayed by two minutes.",
-			"The shuttle's navigation programming was replaced by a file containing two words, IT COMES. The shuttle will be delayed by two minutes.",
-			"The shuttle's custodian tore out his guts and began painting strange shapes on the floor. The shuttle will be delayed by two minutes.",
-			"A shuttle engineer began screaming 'DEATH IS NOT THE END' and ripped out wires until an arc flash seared off her flesh. The shuttle will be delayed by two minutes.",
-			"A shuttle inspector started laughing madly over the radio and then threw herself into an engine turbine. The shuttle will be delayed by two minutes.",
-			"The shuttle dispatcher was found dead with bloody symbols carved into their flesh. The shuttle will be delayed by two minutes.")
+			curses = list("A fuel technician just slit his own throat and begged for death. The shuttle will be delayed by three minutes.",
```
Comment (optimumtact) [inline code/game/gamemodes/cult/cult_items.dm]: Can you put these in a file in the data folder and then load them from file the first time they're needed or something, I'll love you long time.

**Comment by Kromgar** [comment]
Can mesons see cult walls?

**Comment by Robustin** [comment]
Yea I left cult walls alone for now since the cult doesn't "involuntarily" make them. I might give them an overlay too but since cult walls aren't typically seen in the cult's hidden bases, it wasn't a priority for me.

File: code/game/gamemodes/cult/cult_items.dm (hunk 3 above)
Comment (Robustin) [inline code/game/gamemodes/cult/cult_items.dm]: Show me somewhere in the code that already does that.

**Comment by Kromgar** [comment]
Frankly we need a way to get cult walls that isn't spending cult metal because cult walls aren't even as strong as rmetal walls. Which cult metal is made of

**Comment by Kromgar** [comment]
Also clockwork cult walls get disguised by mesons and they voluntarily make those

**Comment by Robustin** [comment]
You seem to forget that Artificers can poop out cult walls every few seconds.

**Pull request merged**